python/fi_instrumentation/otel.py

```
@@ -418,19 +446,194 @@ class BatchSpanProcessor(_BatchSpanProcessor):
+        return f"{span_id:016x}"
+
+    def export(self, spans) -> SpanExportResult:
+        """
+        Exports a batch of spans in JSON format.
+        Args:
+            spans (list): A list of spans to export.
+        Returns:
+            SpanExportResult: Indicates the success or failure of the export.
+        """
+        if not self._endpoint:
+            warnings.warn("gRPC Exporter endpoint is not configured. Cannot export spans.")
```

Ruff (0.11.9)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)